src/main/resources/mcmod.info

```diff
@@ -12,7 +12,7 @@
   "logoFile": "assets/thaumcraftfix/textures/gui/logo.png",
   "screenshots": [],
   "useDependencyInformation": true,
-  "requiredMods": ["forge@[${forgeminversion},)", "thaumcraft@[6.1.BETA26,)", "mixinbooter@[10.0,)"],
+  "requiredMods": ["forge@[${forgeminversion},)", "thaumcraft@[6.1.BETA26,)"],
```

Comment:
This is a problem, because now if someone has neither MixinBooter nor FermiumBooter installed, the Mixins will just silently not work. There needs to be a check for those being installed, and if not, stopping the game from starting. I'd prefer if it showed the nice dirt-background screen Forge does when a dependency was missing to explain the problem, as I don't want to make people read crash logs for basic functionality.